src/video_core/renderer_opengl/renderer_opengl.h

```diff
@@ -59,6 +60,7 @@ class RendererOpenGL : public RendererBase {
     void DrawScreens();
     void DrawSingleScreenRotated(const ScreenInfo& screen_info, float x, float y, float w, float h);
     void UpdateFramerate();
+    void FrameLimiter(float average_frame_time, int frame_limit);
```

Comment:
my initial idea was to use something like `unsigned int average_frame_time, unsigned int target_frame_time` so that you wouldn't have to pass 2 different datatypes here.